api/test/e2e/widgets/base-widgets.spec.ts

```diff
@@ -94,4 +90,65 @@ describe('Base Widgets', () => {
 
     expect(noneFound.body.data).toHaveLength(0);
   });
+
+  it('Should retrieve a widget with its data by its id (indicator)', async () => {
+    // Given
+    const dataSourceManager = testManager.testApp.get(DataSourceManager);
+    await dataSourceManager.loadQuestionIndicatorMap();
```

Comment:
Since this will be a almost-always-needed precondition, we could move it to a specific method in the test manager, or a helper function